woocommerce/woocommerce-gutenberg-products-block
# Remove experimental flag from Cross-Sells block

Fixes #6965

### Note

To prevent that the Cross-Sells block accidentally gets shipped with the WooCommerce Blocks plugin and gets merged into the WooCommerce core, the block had been placed behind an experimental flag. This PR aims to remove the experimental flag, so that the Cross-Sells block becomes available both when using the WooCommerce Blocks and the WooCommerce plugin.

### Testing

1. Run `npm run build`.
2. Replace the content of`blocks.ini` with the following config:
```
woocommerce_blocks_phase = 1
woocommerce_blocks_env = production
```
3. Create a test page and add the Cart block.
4. Verify that the Cross-Sells block is visible on the Cart block both in the editor and in the frontend.

### WooCommerce Visibility
